src/diagnostics/diagnostic-structs.md

```diff
@@ -116,17 +116,17 @@ In the end, the `SessionDiagnostic` derive will generate an implementation of
 impl SessionDiagnostic for FieldAlreadyDeclared {
     fn into_diagnostic(self, sess: &'_ rustc_session::Session) -> DiagnosticBuilder<'_> {
         let mut diag = sess.struct_err_with_code(
-            rustc_errors::DiagnosticMessage::fluent("typeck-field-already-declared"),
+            rustc_errors::DiagnosticMessage::fluent("typeck_field_already_declared"),
```

Comment:
I think this is incorrect now and that the diagnostic derives generate code using the typed identifiers.